app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -308,6 +323,15 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         }
     }
 
+    @VisibleForTesting(otherwise = PROTECTED)
```

Comment:
nit: I don't see this being used for testing: shall we make this method `protected` and skip the annotation?

Reply:
Ah yeah, most of the functions was copied from the existing code for "app_opened" metric. I was wondering if unit testing needs to be part of this PR?